tests/postgres_test.go

```diff
@@ -266,8 +258,6 @@ func TestDeviceAndSubnetNoDeviceID(t *testing.T) {
 // TestOverlappingSubnet verifies that a query for an IP address will return the
 // most specific subnet that matches, as measured by the subnet's netmask length
 func TestOverlappingSubnet(t *testing.T) {
-	t.Parallel()
```

Comment:
question: Why remove the test parallelization?

Reply:
Each one of these tests appear to be modifying the database and then making assertions based on those modifications. They were failing when I pulled down master. Not sure how they were ever passing.